### PR TITLE
feat signup

### PR DESCRIPTION
**Pull request by deepak29-git**
Add signup feat

### CODE REVIEW

File: src/Pages/Signup/Signup.jsx [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        email: user.email,
+        password: user.password,
+      });
+
+      localStorage.setItem("token", response.data.encodedToken);
+
+      setAuth(true);
+      navigate("/login");
+    } catch (error) {
+      console.log(error);
```
Comment (sharathnair9999) [inline src/Pages/Signup/Signup.jsx:57]: you can show error in an  alert if possible

File: src/Pages/Signup/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+          <div className="center">
+            <button onClick={signupHandler} className="my-2 login-btn primary-bg">
+              Create New Account
+            </button>
+          </div>
+          <div className="center">
+            <Link className="black-text" to="/login">
+              Already have an account
+            </Link>
+            <svg
```
Comment (sharathnair9999) [inline src/Pages/Signup/Signup.jsx:151]: you can host the svg and use it for clean code

**Review by sharathnair9999 — APPROVED** [review]

File: src/Pages/Signup/Signup.jsx (hunk 1 above)
Comment (deepak29-git) [reply src/Pages/Signup/Signup.jsx:57]: ok

File: src/Pages/Signup/Signup.jsx (hunk 2 above)
Comment (deepak29-git) [reply src/Pages/Signup/Signup.jsx:151]: ok

**Review by tushargoyal812 — COMMENTED** [review]
code is very clean and to the point.
variable naming is very good.
looks good to merge.

**Pull request merged**